core/src/components/radio-button-group/radio-button-group.stories.js

```diff
@@ -40,17 +42,36 @@ export const Default = () => `
         <input type="radio" class="fudis-radio-button__input" name="fudis-fieldset-id" value="baz" />
         <div class="fudis-radio-button__content">
           <div class="fudis-radio-button__content-wrapper">
-            <div class="fudis-radio-button__content__control"></div>
+            <div class="fudis-radio-button__content__control">
+              <!-- When selected, render the indicator below -->
+              <!-- <span class="fudis-radio-button__content__control__indicator"></span> -->
+            </div>
           </div>
           <span class="fudis-radio-button__label">Option 3</span>
         </div>
       </label>
+      <div id="fudis-guidance-id" class="fudis-guidance">
+        <div>
```

Comment:
Is this extra `div` necessary? It is also in _Guidance_ and _Invalid_ example.

Reply:
Ah, it feels unnecessary since it doesn't have (i.e need) any id in RadioButtonGroup or CheckboxGroup components. For example in TextInput and TextArea components, the inputs are describedby that specific `div` in the Guidance component, whereas CheckboxGroup and RadioButtonGroup are descirbedby the upper `div` element.